web_monitoring/html_diff_render.py

```diff
@@ -810,17 +926,51 @@ def html(self):
 # I had some weird concern that I needed to make this token a single word with
 # no spaces, but now that I know this differ more deeply, this is pointless.
 class ImgTagToken(tag_token):
-    def __new__(cls, tag, data, html_repr, pre_tags=None,
+
+    regex = re.compile(r"(src|srcset)=('|\")(.*?)('|\")")
```

Comment:
I don’t think you actually need this at all (see below), but if you keep it, please use a more descriptive name.